t/08-performance/04-settingkeys.t

```diff
@@ -5,7 +5,7 @@ plan 1;
 my %allowed = (
   Q{!CORE_MARKER},
   Q{Int},
-  Q{&CORE-SETTING-REV},
+  Q{CORE-SETTING-REV},
```

Comment:
Just wondering, but do we need a `!CORE_MARKER` still?  Or vice-verse, couldn't we just put the revision info in `!CORE_MARKER` ?

Reply:
Probably not any more. I wouldn't put version into `!CORE_MARKER` because it would not be directly accessible. Besides, CORE-SETTING-REV allows for testing of constant declaration in COREs.

If no other modules somehow rely on `!CORE_MARKER` I can pull it out later.